client/src/pages/components/EnergisersCard.js

```diff
@@ -1,19 +1,30 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import React, { useState } from "react";
 
-function EnergisersCard({ item }) {
-	return (
-		<Link to={`/energizers/${item.id}`} style={{ textDecoration: "none", color: "inherit" }}>
-			<div className="energizer-box">
-				<section className="energizer">{item.description}</section>
-				<section className="energizer-info-sec">
-					<h3 className="info-sec-name">
-						{item.name}
-					</h3>
-					<p>Rate:{item.rating}</p>
-				</section>
-			</div>
-		</Link >
-	);
-}
-export default EnergisersCard;
+const SearchFilter = ({ energizers }) => {
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const handleSearch = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
+  const filteredEnergizers = energizers.filter((energizer) => {
```

Comment:
I dont think this will work
where are you getting the energizers variable from?
also this component is a child of the Energizers component  and it does something different from what you are trying to do here, it is returning just a single card back to the Energizers component which is rendered out inside the map function

Reply:
All of those you can ignore totally, just two last ones: add new migrations and add url_address in API. These two were on my task. 
I don't know why these old file versions jump together in this pull request and how I remember I wasn't committed.